### DISCUSSION TITLE
[Material property assign on boundary]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#18124] [Q&A Modules: General] [answered]
Dear moose users,

I am trying to find a way to implement material properties only on boundaries (Surfaces or points).

Is there any way to do this?

I found the Interfacematerials.C, but it's for interfaces of two different domains.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can specify the 'boundary' parameter in the same way that you define 'block' for a material defined inside the domain. 

Guillaume

**Reply by coskrrb2002** [reply]
Thank you for your comment @GiudGiud .

It works!

Cheers,
Nakkyu
